presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -53,19 +62,20 @@ public void runCommand()
     {
         switch (packageType) {
```

Comment:
I think this can be improved by Configuration. This should also be done for status